.github/workflows/build_make.yml

```diff
@@ -0,0 +1,210 @@
+name: Build Make
+
+# For any push we try to build the project with make
+# When a PR is open we do more test such as test more make rules
+on: [push,pull_request]
+
+env:
+  OCAML_DEFAULT_VERSION: 4.10.0
+
+jobs:
+  make_all:
+    # Basic test to check if the project build with make.
+    # This test also check that .opam files are synced with dune-project
+    # If this test fails, no more make building test are done
+    name: Make all
+
+    strategy:
+      matrix:
+        cache-name:
+          - make-all
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Retrieve opam cache
+        uses: actions/cache@v2
+        id: cache-opam
+        with:
+          path: ~/.opam
+          key: v1-${{ runner.os }}-alt-ergo-${{ matrix.cache-name }}-${{ env.OCAML_DEFAULT_VERSION }}-${{ hashFiles('*.opam') }}
+
+      - name: Use OCaml ${{ env.OCAML_DEFAULT_VERSION }}
+        uses: avsm/setup-ocaml@v1
+        with:
+          ocaml-version: ${{ env.OCAML_DEFAULT_VERSION }}
+
+      - name: Set git user
+        run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions-bot@users.noreply.github.com
+
+      - name: Pin no action
+        run: opam pin add . -y --no-action
+
+      - name: Install depext
+        run: opam depext -y alt-ergo-lib alt-ergo-parsers alt-ergo altgr-ergo
+        # if: steps.cache-opam.outputs.cache-hit != 'true'
+
+      - name: Install deps
+        run: opam install -y ./*.opam --deps-only --with-test
+        if: steps.cache-opam.outputs.cache-hit != 'true'
+
+      - name: Fixup
+        run: opam upgrade --fixup
+        if: steps.cache-opam.outputs.cache-hit == 'true'
+
+      - name: Make
+        # make use dune build which update .opam file if the dune-project is update
+        run: opam exec -- make
+
+      - name: Check opam files
+        # Check that opam files are unchanged
+        run: |
+          if git diff --exit-code *.opam ; then
+          echo "opam files ok" ;
+          else
+          exit 1 ;
+          fi
```

Comment:
That's a nice addition to the CI. Did you have time to look at github bots that could suggest the diff as changes to be included in the PR ?

Reply:
I've tried to test and use diff-action (https://github.com/marketplace/actions/get-diff-action) on my personnal fork. I was able to easily get the diff for `.opam` files. But the diff was between the `base branch` and the `target branch`. I will try to change the `from and to` https://github.com/technote-space/get-diff-action#from-to

Reply:
> I've tried to test and use diff-action (https://github.com/marketplace/actions/get-diff-action) on my personnal fork. I was able to easily get the diff for `.opam` files. But the diff was between the `base branch` and the `target branch`. I will try to change the `from and to` https://github.com/technote-space/get-diff-action#from-to

This solution did not worked.
I've look at https://github.com/marketplace/actions/action-git-diff-suggestions as suggested. The results are way better.
Note that this action only blocking if a PR is open. I suppose this  is still acceptable ?